### PR TITLE
add zh-Hant(loaclization support)

### PR DESCRIPTION
**Pull request by powenn**
IMG_1779

### CODE REVIEW

File: Taurine/app/zh-Hant.lproj/Main.strings [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+"A2I-I3-Zp4.text" = "碧藍航線";
+
+/* Class = "UITextView"; text = "Log Window Here"; ObjectID = "CQG-gn-Oeq"; */
+"CQG-gn-Oeq.text" = "Log Window Here";
+
+/* Class = "UILabel"; text = "Log Window"; ObjectID = "Goe-t0-Dt2"; */
+"Goe-t0-Dt2.text" = "紀錄視窗";
+
+/* Class = "UILabel"; text = "An update is available at taurine.app"; ObjectID = "HSX-8w-slN"; */
+"HSX-8w-slN.text" = "有可用的更新在taurine.app";
```
Comment (asdfugil) [inline Taurine/app/zh-Hant.lproj/Main.strings:30]:
```suggestion
"HSX-8w-slN.text" = "在taurine.app有可用的更新";
```

File: Taurine/app/zh-Hant.lproj/Main.strings [hunk 2]
```diff
@@ -0,0 +1,114 @@
+
+/* Class = "UILabel"; text = "Odyssey Team"; ObjectID = "0JX-WD-13L"; */
+"0JX-WD-13L.text" = "Odyssey Team";
+
+/* Class = "UILabel"; text = "Enable Tweaks"; ObjectID = "1cs-wY-Xaa"; */
+"1cs-wY-Xaa.text" = "啟用插件";
+
+/* Class = "UILabel"; text = "Restore RootFS"; ObjectID = "20R-01-wfL"; */
+"20R-01-wfL.text" = "回復RootFS";
```
Comment (asdfugil) [inline Taurine/app/zh-Hant.lproj/Main.strings:9]:
```suggestion
"20R-01-wfL.text" = "回復根檔案系統";
```

**Review by asdfugil — CHANGES_REQUESTED** [review]

File: Taurine/app/zh-Hant.lproj/Main.strings (hunk 2 above)
Comment (powenn) [reply Taurine/app/zh-Hant.lproj/Main.strings:9]: I have several questions want to ask ,how can I contact you

Comment (asdfugil) [reply Taurine/app/zh-Hant.lproj/Main.strings:9]: just reply here as far as this pull request goes

Comment (powenn) [reply Taurine/app/zh-Hant.lproj/Main.strings:9]: Alright,first,thanks for your response,and these are my questions:

- seems like you can read Chinese,where you live or come from ?,I live in Taiwan,is a senior high school student(if you mind privacy ,you can skip this )

- I saw you wrote GitHub action to manticore, do you know how to write one to Taurine ? I need a makefile or workflow to build ipa that contain localization changes and installable,I met some problems ,wish you can help me and teach me that (if have extra time)

- I saw you are one of members in procursus and manticore,how did you learn code,program,development,what source you using to learn,

**Pull request closed without merging**